tests/e2e/helm-chart/chainsaw-test.yaml

```diff
@@ -0,0 +1,128 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  name: helm-chart
+spec:
+  description: This e2e test install Odigos via helm chart on custom namespace
+  skipDelete: true
+  steps:
+  - name: Prepare destination
+    try:
+      - script:
+          content: |
+            helm repo add grafana https://grafana.github.io/helm-charts
+            helm install e2e-tests grafana/tempo -n traces --create-namespace --set tempo.storage.trace.block.version=vParquet4
+      - assert:
+          file: assert-tempo-running.yaml
+  - name: Wait for destination to be ready
+    try:
+      - script:
+          timeout: 60s
+          content: ../../common/wait_for_dest.sh
+  - name: Install Odigos
+    try:
+    - script:
+        content: |
+          git clone https://github.com/odigos-io/odigos-charts.git /tmp/odigos-charts
+          # Retry 3 times to avoid flakiness
+          while ! helm upgrade --install odigos /tmp/odigos-charts/charts/odigos --create-namespace --namespace odigos-test-ns --set image.tag=e2e-test; do
+            echo "Failed to install Odigos, retrying..."
+            sleep 1
+          done
+          kubectl label namespace odigos-test-ns odigos.io/system-object="true"
+          rm -rf /tmp/odigos-charts
+        timeout: 60s
+  - name: Verify Odigos Installation
+    try:
+    - script:
+        content: |
+          export ACTUAL_VERSION=$(../../../cli/odigos version --cluster)
+          if [ "$ACTUAL_VERSION" != "e2e-test" ]; then
+            echo "Odigos version is not e2e-test, got $ACTUAL_VERSION"
+            exit 1
+          fi
+    - assert:
+        file: assert-odigos-installed.yaml
+  - name: Install Demo App
+    try:
+    - script:
+        timeout: 100s
+        content: |
+          docker pull keyval/odigos-demo-inventory:v0.1
```

Comment:
I think this sure work:
`kubectl apply -f https://raw.githubusercontent.com/odigos-io/simple-demo/main/kubernetes/deployment.yaml`
Then we don't need to remember to update versions.

Reply:
Applying the remote YAML file does two things: pull all the images (because it's a fresh cluster) and run the deployments.
Because the GitHub action runners sometimes have bad internet connection it's hard to predict how much timeout we should put for both pulling images + running apps so I decided to split it into two steps for stability.